Logger.go

```diff
@@ -235,3 +251,17 @@ func actualValue(value interface{}) interface{} {
 
 	return value
 }
+
+const requestId = "x-request-id"
+
+type keyType int
+
+const key keyType = 0
+
+func FromRequest(req *http.Request) string {
```

Comment:
bij `logger.FromRequest()` zou ik een logger instance terug verwachten. Dat is niet wat je hier op doelt geloof ik.